src/main/java/io/jenkins/plugins/intotorecorder/InTotoRecorder.java

```diff
@@ -149,7 +149,7 @@ public boolean prebuild(AbstractBuild<?,?> build, BuildListener listener)  {
         this.cwd = build.getWorkspace();
         String  cwdStr;
         if (this.cwd != null) {
-            cwdStr = this.cwd.toString();
+            cwdStr = this.cwd+"";
```

Comment:
Is this the canonical way to make this type casting? I don't see a replacement method in Hudson, so I suppose it's okay.

Reply:
well, this is not exactly casting as far as I'm aware. I'm surprised they deprecated toString(). can we verify this returns the actual path and not some wonky retval like "FilePath@1234134" ?

Reply:
@lakshya8066 bump! I think we should get this merged.